### PR TITLE
CONTRAST-27030 cef is now security; add metadata; other cleanup

### PR DESCRIPTION
**Pull request by palmergs**
I think the primary changes for common config for ruby and python were to update cef to security logger and standardize some of the terminology.

### CODE REVIEW

**Comment by kaitlynmortimer** [comment]
@palmergs I'm changing the base to my current branch for all Common Config docs, which matches my Bitbucket branch (with the same name) for page/side nav updates. The current branch also includes a new, separate article for each agent's full YAML template.

File: content/installation/python/PythonConfig.md [hunk 1]
```diff
@@ -11,40 +11,136 @@ The Python agent and Contrast Service use a YAML file to alter the agent behavio
...
+
+### Logger
+
+Define the following properties to set logging values. If these properties aren't defined, the agent uses the logging values from the Contrast UI.
+
+* **agent**: Options for communicating between the agent and the service
+  * **logger**:
+    * **path**: Enable diagnostic logging by setting a path to a log file. While diagnostic logging hurts performance, it generates useful information for debugging Contrast. The value set here is the location to which the agent saves log output. If no log file exists at this location, the agent creates a file. <br> Example - */opt/Contrast/contrast.log* creates a log in the */opt/Contrast* directory, and rotates it automatically as needed.
+    * **level**: Set the the log output level. Value options are `OFF`, `FATAL`, `ERROR`, `WARN`, `INFO`, `DEBUG`, `TRACE`, and `ALL`.
     * `progname`: Name used to identify the process within the log file (Contrast Agent)
```
Comment (kaitlynmortimer) [inline content/installation/python/PythonConfig.md]: @palmergs I don't see this property listed in the YAML template for Python. Has this been added very recently?

File: content/installation/python/PythonConfig.md [hunk 2]
```diff
@@ -11,40 +11,136 @@ The Python agent and Contrast Service use a YAML file to alter the agent behavio
...
+  * **port**: Set the the port of the Contrast service to which the Contrast agent should report. **Required.** <br> Example: `30555`
+    
+#### Logger
+
+The following properties are used by the logger in the Contrast service. If the properties are not defined, the service uses the logging values from the Contrast UI.
+
+  * **logger**:
+    * **path**: Set the location to which the Contrast service saves log output. If no log file exists at this location, the service creates one. <br> Example: */opt/Contrast/contrast_service.log* will create a log in the */opt/Contrast* directory.
+    * **level**: Set the the log output level. Options are `OFF`, `FATAL`, `ERROR`, `WARN`, `INFO`, `DEBUG`, `TRACE`, and `ALL`.
+    * `progname`: Name to identify the process with the service log (Contrast Service)
```
Comment (kaitlynmortimer) [inline content/installation/python/PythonConfig.md]: @palmergs Same here.

File: content/installation/python/PythonConfig.md (hunk 1 above)
Comment (palmergs) [reply content/installation/python/PythonConfig.md]: `progname` is used in the python agent. Are you saying it was missed in this update?

**Pull request merged**